docker/docker-compose.integration-mysql.yml

```diff
@@ -1,8 +1,7 @@
 services:
   mysql_example:
-    image: mysql
+    image: mysql:9
```

Comment:
i'd actually stray away from pinning this, i think (assuming that doesn't break CI...if it did, that'd surprise me!)

the reason is that it's helpful to actually pull in the latest and greatest, it's a way to flag to us if our current integration support is not working -- for whatever reason -- with the latest version of the db. it's obviously inconvenient when CI breaks unexpectedly from underneath us, but in my view, that's a helpful defense mechanism against us accidentally making assumptions that don't hold with new versions of the software we're integrating with.

what do you think?

Reply:
I tend to lie on the side of pinning dependencies in tests just so we don't have something like exactly this where something fails (not silently, but quietly maybe 😄) over a holiday week and isn't caught. But it looks like integration tests don't fail builds/block releases so maybe the failure is a better indicator in a case like this, that we need to update tests. I'll defer to you, no strong feelings.

Reply:
Actually I revise what I said before, I think that we should be pinning to _something_ as a version, but it doesn't perturb me to pin to `latest` or something else that would allow updates. Do you know what we're getting when we just leave off a version? I'll look that up, I don't know if it switched to v9 last week as a `latest` release or `stable` or something else

Reply:
pinned to `latest` here: https://github.com/ethyca/fides/pull/5070/commits/8d2f89f7e9d0ab102332dab7168cf12f20b41ae6